integration-commons/src/main/java/org/symphonyoss/integration/event/HealthCheckServiceEvent.java

```diff
@@ -0,0 +1,19 @@
+package org.symphonyoss.integration.event;
+
+/**
+ * Event object to perform the health check in an specific service.
+ *
+ * Created by rsanchez on 22/03/17.
+ */
+public class HealthCheckServiceEvent {
```

Comment:
What makes this an Event?  Same with the following class MessageMLVersionUpdatedEvent. They don't implement an Event interface.  They just seem to be wrapping a single field.